All EIPIP Meetings/Meeting 060.md

```diff
@@ -0,0 +1,121 @@
+
+**Micah:** a similar note that is license for a reference implementation discussion so um on this 157 issue there's two separate things one is should we require everything to be a certain license or not and two should we lint people's code i'm pretty against linking people's code because i want authors to have autonomy to write their code how they want and if you want to do curly braces on next line with you know vertical alignment and using spaces then that's your choice and i don't want to try to enforce on anyone else any particular um this is mainly because i don't want editors to be taking philosophical positions on things at all possible so that's the one thing i'm fairly
```

Comment:
```suggestion
**Pooja:**  i see point number five which is also on a similar note that is license for a reference implementation discussion 

**Micah**: so um on this 157 issue there's two separate things one is should we require everything to be a certain license or not and two should we lint people's code i'm pretty against linking people's code because i want authors to have autonomy to write their code how they want and if you want to do curly braces on next line with you know vertical alignment and using spaces then that's your choice and i don't want to try to enforce on anyone else any particular um this is mainly because i don't want editors to be taking philosophical positions on things at all possible. 
```